build.gradle

```diff
@@ -276,6 +276,8 @@ dependencies {
   testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: versions.junit
 
   testCompile group: 'org.springframework.boot', name: 'spring-boot-starter-test'
+  testCompile "com.github.tomakehurst:wiremock:2.19.0"
```

Comment:
Not sure why wiremock is needed you can achieve the same by just using SpringbootTEst

Reply:
I used the ia-case-api as the base. I am imitating an external API built by HomeOffice and it is essential to mock the response.